packages/connect-query/src/call-unary-method.ts

```diff
@@ -18,36 +18,32 @@ import type {
   MessageShape,
 } from "@bufbuild/protobuf";
 import { create } from "@bufbuild/protobuf";
-import type { CallOptions, Transport } from "@connectrpc/connect";
+import type { Transport } from "@connectrpc/connect";
 
 import type { MethodUnaryDescriptor } from "./method-unary-descriptor.js";
 
 /**
  * Call a unary method given its signature and input.
  */
+// eslint-disable-next-line @typescript-eslint/max-params -- 4th param is optional
 export async function callUnaryMethod<
   I extends DescMessage,
   O extends DescMessage,
 >(
+  transport: Transport,
```

Comment:
Removing `CallOptions` is a breaking change, but we're also changing the function signature here. Transport is strictly necessary to make a call, so I think it's a better fit to let it be a required argument than part of an options argument.